cs-excel/src/main/java/io/cloudslang/content/excel/utils/Descriptions.java

```diff
@@ -0,0 +1,128 @@
+        public static final String ROWS_COUNT_DESC = "The number of the row indexes returned.";
+        public static final String SUCCESS_DESC = "The row indexes were retrieved successfully.";
+        public static final String FAILURE_DESC = "Failed to retrieve the row indexes.";
+        public static final String EXCEPTION_DESC = "An error message in case there was an error while retrieving the row data.";
+    }
+
+    public static class CreateExcelFile {
+        public static final String WORKSHEET_NAMES_DESC = "A delimited list of names that will be created as worksheets in the document.\n" +
+                "Default value: Sheet1,Sheet2,Sheet3.";
+        public static final String DELIMITER_DESC = "The character used to delimit worksheet names.\n" +
+                "Default value: comma (,)";
+        public static final String RETURN_RESULT_DESC = "This is the primary output. Returns a success or failure message.\n";
+        public static final String EXCEPTION_DESC = "An error message in case there was an error creating the file.";
+        public static final String SUCCESS_DESC = "A new Excel document is created successfully.";
+        public static final String FAILURE_DESC = "Failed to create a new Excel document.";
+    }
+
+    public static class DeleteCell {
+        public static final String ROW_INDEX_DESC = "A list of row indexes.\n" +
+                "Examples: 1:3, 10, 15:20,25";
+        public static final String COLUMN_INDEX_DESC = "A list of column indexes\n" +
+                "Examples: 1:3, 10, 15:20,25";
+        public static final String RETURN_RESULT_DESC = "This is the primary output. Returns the number of rows that were affected.";
+        public static final String SUCCESS_DESC = "The cell data specified was deleted successfully.";
+        public static final String FAILURE_DESC = "Failed to delete the cell data specified";
+        public static final String EXCEPTION_DESC = "An error message in case there was an error while deleting the cell.";
+    }
+
+    public static class AddCell {
+        public static final String HEADER_DATA_DESC = "A delimited list of column names. If left blank, the document will not have a header for the data.";
+        public static final String ROW_DATA_DESC = "A delimited list of data.";
+        public static final String ROW_DELIMITER_DESC = "The delimiter used to separate the rows of the returnResult.\n" +
+                "Default value: | (pipe)";
+        public static final String COLUMN_DELIMITER_DESC = "The delimiter used to separate the columns of the returnResult.\n" +
+                "Default value: , (comma)";
+        public static final String ROW_INDEX_DESC = "A list of row indexes.\n" +
+                "Examples: 1:3, 10, 15:20,25\n" +
+                "Default Value: from the index of the first row  to the index of the last row in the Excel worksheet.";
```

Comment:
Is last row in excel or it's last row with a value?

Reply:
Is last row in excel